<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>CategoryEditDialog_base</class>
 <widget class="QWidget" name="CategoryEditDialog_base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit Categories</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" colspan="2" >
    <widget class="QLineEdit" name="mEdit" />
   </item>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="KPushButton" name="mButtonAdd" >
       <property name="text" >
        <string>A&amp;dd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="mButtonAddSubcategory" >
       <property name="text" >
        <string>Add &amp;Subcategory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="mButtonRemove" >
       <property name="text" >
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>50</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="KPIM::AutoCheckTreeWidget" name="mCategories" >
     <property name="autoCheck" >
      <bool>false</bool>
     </property>
     <property name="dragDropMode" >
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <property name="columnCount" >
      <number>1</number>
     </property>
     <column>
      <property name="text" >
       <string>Test</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="KDialog::spacingHint()" margin="KDialog::marginHint()" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KPIM::AutoCheckTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>autochecktreewidget.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
    <class>KPushButton</class>
    <extends>QPushButton</extends>
    <header>KPushButton</header>
    <container>0</container>
    <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mCategories</tabstop>
  <tabstop>mEdit</tabstop>
  <tabstop>mButtonAdd</tabstop>
  <tabstop>mButtonAddSubcategory</tabstop>
  <tabstop>mButtonRemove</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
